src/config/schema.ts

```diff
@@ -0,0 +1,339 @@
+      format: 'int',
+      default: 100,
+      env: 'SES_MAX_MESSAGES',
+    },
+    maxConnections: {
+      doc: 'Connection pool to send email in parallel to the SMTP server',
+      format: 'int',
+      default: 38,
+      env: 'SES_POOL',
+    },
+    socketTimeout: {
+      doc: 'Milliseconds of inactivity to allow before killing a connection',
+      format: 'int',
+      default: 600000,
+      env: 'MAIL_SOCKET_TIMEOUT',
+    },
+  },
+  awsConfig: {
+    region: {
+      doc: 'Region that S3 bucket is located in',
+      format: Object.keys(awsInfo.data.regions),
+      default: AWS_DEFAULT.region,
+      env: 'AWS_REGION',
+    },
+    customCloudWatchGroup: {
+      doc:
+        'Name of CloudWatch log group to store short-term logs. Log streams are separated by date.',
+      format: String,
+      default: '',
+      env: 'CUSTOM_CLOUDWATCH_LOG_GROUP',
+    },
+  },
+  core: {
+    port: {
+      doc: 'Application Port',
+      format: 'port',
+      default: 5000,
+      env: 'PORT',
+    },
+    otpLifeSpan: {
+      doc:
+        'OTP Life Span for Login. (Should be in miliseconds, e.g. 1000 * 60 * 15 = 15 mins)',
+      format: 'int',
+      default: 900000,
+      env: 'OTP_LIFE_SPAN',
+    },
+    submissionsTopUp: {
+      doc: 'Number of submissions to top up submissions statistic by',
+      format: 'int',
+      default: 0,
+      env: 'SUBMISSIONS_TOP_UP',
+    },
+    nodeEnv: {
+      doc: 'Express environment mode',
+      format: [Environment.Prod, Environment.Dev, Environment.Test],
+      default: Environment.Prod,
+      env: 'NODE_ENV',
+    },
+  },
+}
+
+export const prodOnlyVarsSchema: Schema<IProdOnlyVarsSchema> = {
+  port: {
+    doc: 'SMTP port number',
+    format: 'port',
+    default: null,
+    env: 'SES_PORT',
+  },
+  host: {
+    doc: 'SMTP hostname',
+    format: String,
+    default: null,
+    env: 'SES_HOST',
+  },
+  user: {
+    doc: 'SMTP username',
+    format: String,
+    default: null,
+    env: 'SES_USER',
+  },
+  pass: {
+    doc: 'SMTP password',
+    format: String,
+    default: null,
+    env: 'SES_PASS',
+  },
+  dbHost: {
+    doc: 'Database URI',
+    format: (val) => {
+      // Will throw error if scheme and hosts are not present
+      const uriObject = mongodbUri.parse(val)
+      /*
+        e.g. mongodb://database:27017/formsg will be parsed into:
+        {
+          scheme: 'mongodb',
+          database: 'formsg',
+          hosts: [ { host: 'database', port: 27017 } ]
+        }
+        e.g. https://form.gov.sg will be parsed into: 
+        { 
+          scheme: 'https', 
+          hosts: [ { host: 'form.gov.sg' } ] 
+        }
+      */
+      if (uriObject.scheme !== 'mongodb') {
+        throw new Error('Scheme must be mongodb')
+      }
+      if (isNil(uriObject.database)) {
+        throw new Error('Database must be specified')
+      }
+    },
+    default: null,
+    env: 'DB_HOST',
```

Comment:
I'm wondering whether we should also be adding `sensitive: true` here, so we can safely call `config.toString()`

Reply:
done

Reply:
are there any other sensitive env vars?